src/fillTank.test.js

```diff
@@ -1,11 +1,85 @@
 'use strict';
 
 describe('fillTank', () => {
-  // const { fillTank } = require('./fillTank');
+  const { fillTank } = require('./fillTank');
 
-  it('should ', () => {
+  it(`should be declared`, () => {
+    expect(fillTank).toBeInstanceOf(Function);
+  });
+
+  it('should be fill all tunk', () => {
```

Comment:
Typo in the test description: 'should be fill all tunk' should be corrected to 'should fill the entire tank'.